### DISCUSSION TITLE
Minor bug with DigestAuth and key types

### DISCUSSION THREAD

**Discussion by x-stride** [encode/httpx#2192] [Potential Issue] [answered]
Hello all,

Spotted around [here](https://github.com/encode/httpx/blob/b07fe7b0745e62be5ef9bce1bee9e7d7a8878552/httpx/_auth.py#L142)
Some possibly outdated servers respond with hash/key types in lowercase - the function assume uppercase...

:)

**Comment by lovelydinosaur** [comment, marked as answer]
Yup, seems like a valid improvement to make there.
Addressed as pull request https://github.com/encode/httpx/pull/2204.

Thanks!

**Reply by x-stride** [reply]
Nice work!
